benchmark/scripts/prepare_configs.py

```
@@ -287,95 +300,110 @@ def cliproxy_config(
+    default level set is low/medium/high, so a model whose levels omit `max`
+    would silently downgrade PA1's `max` requests to `high`. Rendering the
+    expected block from the same catalog entry the Codex side uses, and
+    requiring the template to contain it verbatim, means the two cannot drift.
+    """
+    for entry in entries:
+        block = "\n".join(cliproxy_model_block(entry))
+        if block not in template:
+            raise SystemExit(
+                f"{path}: the block for {entry['slug']!r} does not match the "
+                f"frozen Codex catalog. Expected to find:\n{block}"
+            )
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)